index.html

```diff
@@ -15,9 +16,10 @@ <h2 class="card-title">Welcome to todo list app</h2>
           <div class="form-group">
             <input type="text" class="form-control" id="text" placeholder="input text">
           </div>
-          <button type="submit" class="btn btn-primary" id="button">Submit</button>
+          <button type="submit" class="btn btn-primary" onclick="myFunction()" id="button">Submit</button>
         </div>
       </div>
+      <div id="tag"></div>
```

Comment:
Tag has very wide sense. Try always name with some close meaning (for example todo-list, wrapper, etc..) . As this is a wrapper for a list, why not to use <ul>?

Reply:
ok, i fixed this